assets/main.js

```diff
@@ -932,17 +932,51 @@ async function updateToken(value) {
 	}
 }
 
-async function updateUnikernel(job, latest_build, current_build, unikernel_name) {
+async function updateUnikernel(job, to_be_updated_unikernel, currently_running_unikernel, unikernel_name) {
```

Comment:
I'm not sure why these renamings are part of this PR. Makes it rather hard to review, but so be it.

Reply:
no need to act on this comment, but in the future it would be great if such things would be in a separate PR.